lte/gateway/python/magma/kernsnoopd/ebpf/packet_count.bpf.c

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This program attaches to the packet transmit event net_dev_start_xmit. It
+ * counts the number of bytes and packets sent by each linux binary.
+ * Expect considerable performance impact.
+ * https://patchwork.ozlabs.org/patch/309437/
+ */
+
+// Create a hash map with `key_t` as key type and `counters_t` as value type
+BPF_HASH(dest_counters, struct key_t, struct counters_t, 1000);
+
+// Attach hook for the `net_dev_start_xmit` kernel trace event
+TRACEPOINT_PROBE(net, net_dev_start_xmit) {
```

Comment:
since this is hooking up to all L2 traffic,
can you post perf numbers from the lab spirent setup, for scale test like, 600 UE @ 5 mbps DL traffic for each UE.

Reply:
Done.